frontend/app/src/modules/dashboard/graph/use-net-value-event-handlers.ts

```diff
@@ -179,6 +213,22 @@ export function useNetValueEventHandlers(params: UseNetValueEventHandlersParams)
     });
   }
 
+  function setupZoomToolHandler(instance: EChartsType): void {
+    const activateZoomTool = (): void => {
+      // @ts-expect-error accessing private property
+      if (instance._componentsViews[0]._features.dataZoom._isZoomActive) {
```

Comment:
no other way to track this without relying on this structure? it seems hacky and like something that would easily break

Reply:
I can try to find another way, but I can't guarantee that I will find one. I just realized their API is not very good to do simple thing like this :(